__tests__/utils.tsx

```diff
@@ -146,21 +163,28 @@ test('NotFoundResponse', async () => {
 
 describe('fetchWithCache()', () => {
   test('returns the result of fetch()', async () => {
-    mockFetch({ 'http://example.com/': 'test' })
+    server.use(
+      rest.get('http://example.com/', (_req, res, ctx) => {
+        return res.once(ctx.status(200), ctx.body('test'))
+      })
+    )
 
     const response = await fetchWithCache('http://example.com/')
 
     expect(await response.text()).toBe('test')
   })
 
   test('responses are cached for 1 hour', async () => {
-    const mockedFetch = mockFetch({ 'http://example.com/': '' })
-
-    await fetchWithCache('http://example.com/')
-
-    expect(mockedFetch.getCallArgumentsFor('http://example.com/')).toEqual([
-      'http://example.com/',
-      { cf: { cacheTtl: 3600 } },
-    ])
+    let request!: MockedRequest
+    server.use(
+      rest.get('http://example.com/', (_req, res, ctx) => {
+        request = _req
+        return res.once(ctx.status(200), ctx.body('test'))
+      })
+    )
+
+    const fetch = jest.fn().mockResolvedValueOnce(new Response(''))
+    await fetchWithCache('http://example.com/', undefined, fetch)
+    expect(fetch).toBeCalledWith(request)
```

Comment:
```suggestion
    expect(fetch).toBeCalledWith("http://example.com/", {cf: { cacheTtl: 3600 }})
```

Reply:
Here you need to check whether the given fetch function was actually called with the arguments `"http://example.com/", {cf: { cacheTtl: 3600 }}`